src/eredis.app.src

```diff
@@ -1,6 +1,6 @@
 {application,eredis,
              [{description,"Erlang Redis Client"},
-              {vsn,"1.3.1-nordix"},
+              {vsn,"1.3.1"},
```

Comment:
I think we cannot use 1.3.1 now after 1.3.2-nordix is published.... I guess we must use 1.3.2 (or something else...)